recipe/meta.yaml

```diff
@@ -32,7 +34,7 @@ requirements:
     - python             # [win]
     # required for win32: https://wiki.openssl.org/index.php/Compilation_and_Installation#Windows
     - nasm               # [win32]
-    - perl
+    - perl ==5.18.2
```

Comment:
Will need to be at least 5.20.3.1. I don't think we ever had `perl` 5.18. Please re-render after correcting this.